### ISSUE TITLE
Telescope stopped working with packer?

### ISSUE THREAD

**Issue by Indoors-Cat** [nvim-telescope/telescope.nvim#2193]
### Description

Been using Telescope for about a month, installed through packer (the AUR package version). Went to add a new plugin today, plugin wouldn't work, reverted to state where my config was working properly, and now my config only works if I comment out all telescope plugins. 

### Neovim version

```markdown
NVIM v0.8.0
Build type: Release
LuaJIT 2.1.0-beta3
```

### Operating system and version

Arch Linux

### Telescope version / branch / rev

telescope 0.1.0

### checkhealth telescope

```markdown
ERROR: No healthcheck found for "telescope" plugin.
```

### Steps to reproduce

add 
use {
  'nvim-telescope/telescope.nvim', tag = '0.1.0',
-- or                            , branch = '0.1.x',
  requires = { {'nvim-lua/plenary.nvim'} }
} 
to config. 

### Expected behavior

Save file, run :PackerSync & :PackerUpdate, then use telescope as normal to find files.

### Actual behavior

Lua config breaks, disables all plugins, and packer cannot seem to find telescope to install it again. 

### Minimal config

```Lua
vim.cmd [[set runtimepath=$VIMRUNTIME]]
vim.cmd [[set packpath=/tmp/nvim/site]]
local package_root = '/tmp/nvim/site/pack'
local install_path = package_root .. '/packer/start/packer.nvim'
local function load_plugins()
  require('packer').startup {
    {
      'wbthomason/packer.nvim',
      {
        'nvim-telescope/telescope.nvim',
        requires = {
          'nvim-lua/plenary.nvim',
          { 'nvim-telescope/telescope-fzf-native.nvim', run = 'make' },
        },
      },
      -- ADD PLUGINS THAT ARE _NECESSARY_ FOR REPRODUCING THE ISSUE
    },
    config = {
      package_root = package_root,
      compile_path = install_path .. '/plugin/packer_compiled.lua',
      display = { non_interactive = true },
    },
  }
end
_G.load_config = function()
  require('telescope').setup()
  require('telescope').load_extension('fzf')
  -- ADD INIT.LUA SETTINGS THAT ARE _NECESSARY_ FOR REPRODUCING THE ISSUE
end
if vim.fn.isdirectory(install_path) == 0 then
  print("Installing Telescope and dependencies.")
  vim.fn.system { 'git', 'clone', '--depth=1', 'https://github.com/wbthomason/packer.nvim', install_path }
end
load_plugins()
require('packer').sync()
vim.cmd [[autocmd User PackerComplete ++once echo "Ready!" | lua load_config()]]
```

**Comment by Conni2461** [comment]
Could you share the error message? What does installed through packer (the AUR package version) mean? I left arch a couple of years ago, so i am not up to date.

Does telescope work if you go with the minimal config?

So make a new file `min.lua` with the content and run neovim with `nvim -nu min.lua`. Wait a second because we need to install telescope and then test with `:Telescope find_files`

**Comment by Indoors-Cat** [comment]
When i installed packer to manage my plugins, i followed the quickstart guide on the github page:
(git clone --depth 1 https://github.com/wbthomason/packer.nvim\
 ~/.local/share/nvim/site/pack/packer/start/packer.nvim) 
but had issue with it not working. They have listed that they have an AUR package (nvim-packer-git) so I installed that and packer worked immediately for me. 

Telescope didn't work the first time(Perhaps I did not wait long enough?), but now appears to be functioning as expected within the minimal config.

**Comment by Conni2461** [comment]
I would not install it with an aur package. packer.nvim can self update itself. `-git` aur packages need to manually updated and people tend to forget this.

So if i were you i would uninstall that aur package and try again installing it as described in the readme. I use packer like that and i dont experience this issue.

**Comment by Indoors-Cat** [comment]
So after a lot of testing and trying of different things, I have managed to get everything working properly. Here's what I've found:

I still have found better luck using the AUR package to install packer (I do tend to keep my packages fairly up-to-date) for some reason that I have not managed to solve as of yet. 

My main issue I believe was that I misread (perhaps read too quickly) the documentation for packer itself. I was using :PackerInstall to try and install new plugins, which did work, but when I would add more plugins it would remove the directories for those original plugins, as well as the directory for packer.nvim itself. I had this issue using :PackerSync as well originally. I'm not fully certain what happened, as I uninstalled & reinstalled packer numerous times, using both the AUR package, as well as my attempts to use the git clone, but as of this point, everything appears to be functioning as expected. 

TLDR; Works now. Probably user error. Make sure to read the documentation. Thank you for the assistance.
